### PR TITLE
fix(ci): make logicaltest ci cover native format

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

**Comment by sundy-li** [comment]
```
MySQL [(none)]> SELECT b, SUM(a) AS sum FROM test GROUP BY b HAVING b=21 ORDER BY b;
ERROR 1105 (HY000): Code: 1068, displayText = Cannot join handle from context's runtime, cause: task 333 panicked.
```

`b` is binded as `group_item` now,  but it's pushed into storage.  cc @xudong963  @leiysky .

**Review by Xuanwo — APPROVED** [review]

File: native.toml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Usage:
+# databend-query -c databend_query_config_spec.toml
```
Comment (Xuanwo) [inline native.toml:2]: I think this file should not be placed in the root directory.

Comment (sundy-li) [reply native.toml:2]: Oh, i forgot to remove this file.

**Comment by sundy-li** [comment]
Timeout is not related to this pr, let's merge.

**Pull request merged**